### DISCUSSION TITLE
What kind of data does PiVPN collect on the server?

### DISCUSSION THREAD

**Discussion by AdanC1b3r** [pivpn/pivpn#1594] [Q&A]
There are three types of data that VPN servers can log: activity logs, connection logs, and aggregate logs. It's critical to understand what types of data fall into each of these categories to effectively protect your privacy!

**Comment by coolapso** [comment]
I think you are misunderstanding pivpn and its use case! 

PiVPN? collects no data! 

The VPN Protocols that PiVPN installs on your hardware? I don't recall having any tweaked settings as far as logging is concerned so its the default logging level of  each protocol.

Converting to discussion. not an issue!

**Reply by AdanC1b3r** [reply]
So PiVPN is truly logless? does not collect or store any activity or connection data that can be used to identify you personally. More importantly, it won't collect or save any information transmitted through the VPN tunnel, whatever it is.

**Reply by TinCanTech** [reply]
Pivpn is an installation utility. It has no method or reason to **_spy_** on you.

If you have serious security concerns then the place to ask is with either OpenVPN or Wireguard .. or your distro support channels.

**Reply by AdanC1b3r** [reply]
I refer to this why otherwise using the VPN would only be effective to access your resources within the same network, otherwise for privacy and tracking protection it would not be suitable.

**Reply by TinCanTech** [reply]
I don't believe you understand what a VPN does or is used for. I think you only understand the internet Hype from **liars** like NordVPN.

**Reply by AdanC1b3r** [reply]
What the VPN is used for?

**Reply by coolapso** [reply]
>So PiVPN is truly logless? does not collect or store any activity or connection data that can be used to identify you personally. More importantly, it won't collect or save any information transmitted through the VPN tunnel, whatever it is.

1) PiVPN is not a VPN nor a VPN provider nor a VPN Service
2) as @TinCanTech Mentioned, PiVPN is an Installation Utility. Its a way to help less technically inclined users to Install a VPN and keep their privacy on Public networks or access their home network. Its not the goal of PiVPN to provide anonymity from your ISP or to hide your activity from whoever you might fear! For that... I am sorry to disappoint you you will need more than a VPN! 
3) Last but not the least PiVPN does not claim anywhere on the website or documentation to be logless!

**Reply by coolapso** [reply]
> What the VPN is used for?

To provide secure access to remote networks! as simple as that!

**Reply by AdanC1b3r** [reply]
Excellent, that's right

**Reply by coolapso** [reply]
I can recommend you to read this: https://4s3ti.net/en/posts/vpnusage/  might help you understanding a bit more about VPNs and what the VPN Hype is all about!  ;)

**Reply by coolapso** [reply]
> I don't believe you understand what a VPN does or is used for. I think you only understand the internet Hype from **liars** like NordVPN.

The hype is real and people don't even realize they are purely shifting trust from One evil to the Other that they know even less about!

**Reply by AdanC1b3r** [reply]
Thank you for sharing and for devoting a little of your time to this conversation. @4s3ti @TinCanTech

**Reply by coolapso** [reply]
No worries!! Our mission is to educate as well! , hope helped to clear some misconceptions.

**Reply by TinCanTech** [reply]
> The hype is real 

I hope that we can agree to disagree on that. 

From my point of view, **_Hype_** is not real, that is why it only works on people who don't understand the **_real_** problem.

**Reply by coolapso** [reply]
@TinCanTech you could not be more accurate there! and that's what I meant, probably I should have ended with a /s there and started a new line after the and  ;)

**Reply by AdanC1b3r** [reply]
> No worries!! Our mission is to educate as well! , hope helped to clear some misconceptions. 

I don't think so, but I respect your point of view, I am cybersecurity with 43 security certifications in different areas.

**Reply by TinCanTech** [reply]
It is not the **_quantity_** that counts, it is the **_quality_** of such certification.

**Reply by coolapso** [reply]
>  I am cybersecurity with 43 security certifications in different areas. 

Sorry if this sounds rude! but it only says you know how to study for exams :)

**Reply by AdanC1b3r** [reply]
He credits his knowledge in the area, so he says.

**Reply by TinCanTech** [reply]
> He credits his knowledge in the area, so he says.

**_He_** who ? Communications is not your strong suit.

**Reply by TinCanTech** [reply]
And there is a pyramid on the dollar bill .. remember Jeffry Epstein.

**Comment by TinCanTech** [comment]
I just want to make one further comment:

Microsoft own github and Microsoft have **The Anti-trust lawsuit** to answer for, which they have never answered for, even in court.

Therefore, anybody with _serious paranoid illusions_ about their internet privacy, **absolutely** should never use github.
